tests/e2e/velero_yaml_suite_test.go

```diff
@@ -0,0 +1,40 @@
+package e2e
+
+import (
+	"path/filepath"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+type veleroTemplateParams struct {
+	VeleroName      string
+	BslRegion       string
+	S3Url           string
+	CredentialsName string
+	BucketName      string
+}
+
+var _ = AfterEach(func() {
+	// Delete Velero CR once the test has finished
+	err := vel.Delete()
+	Expect(err).ToNot(HaveOccurred())
+})
+
+var _ = Describe("Test Velero CR creation with", func() {
+	It("One Backup Storage Location templated from yaml", func() {
+		// Create Velero CR and verify it is successful
+		veleroTemplate, _ := filepath.Abs("templates/velero_bsl_template.tmpl")
+		params := veleroTemplateParams{
+			"example-vel",
+			"us-west-1",
+			"http://192.156.2.4:9000",
```

Comment:
@nitishSr I don't think we want this value here. Will fail in CI env

Reply:
But I really like the way we are reading from yaml and I would like to update the test suite to use this.

Reply:
Thank you for review @dymurray , I have updated the test with changes.